Update faq.json

jrduscher opened the pull request:
suppression des références à  des tests ou des soucis de validité dans le texte. il peut encore en rester... :\

jrduscher committed:
Update faq.json

suppression des références à  des tests ou des soucis de validité dans le texte. il peut encore en rester... :\

douglasduteil commented on packages/code-du-travail-data/dataset/faq.json:
```diff
@@ -1,26 +1,16 @@
       "themes": [20400],
       "theme": ["durée du travail", "durée maximale quotidienne de travail"],
-      "branche": "TEST"
+      "branche": "";
```
You can't use `;` here. 
You should use `,`

douglasduteil commented on packages/code-du-travail-data/dataset/faq.json:
```diff
@@ -1,26 +1,16 @@
       "theme": ["durée du travail", "durée maximale quotidienne de travail"],
-      "branche": "TEST"
+      "branche": "";
+      "commentaire": "contenu équipe projet";
```
You can't use `;` here. 
You should use `` as it's the last one

douglasduteil commented on packages/code-du-travail-data/dataset/faq.json:
```diff
@@ -916,7 +906,7 @@
   {
     "question": "Quelle est la durée d’un accord de performance collective? Qu’advient-il des clauses suspendues du contrat de travail lorsque l’accord prend fin ? ",
-    "reponse": "réponse non validée",
+    "reponse": "La question fait encore l'obje de débats, et l'une des pistes de réponse à l'étude est la suivante: L’APC est un accord de droit commun s’agissant des règles relatives à sa durée : il peut donc être à durée déterminée ou indéterminée, et à défaut de précision, il est conclu implicitement pour une durée de 5 ans. Les clauses du contrat de travail contraires à l’accord sont suspendu ...
```
Strange `’`...
Should be a `'` I think

douglasduteil commented on packages/code-du-travail-data/dataset/faq.json:
```diff
@@ -982,7 +972,7 @@
   {
     "question": "Quelles sont les conditions de validité de l’accord de performance collective?",
-    "reponse": "<p>Les conditions de validité de l’accord de performance collective sont identiques à celles de tout accord collectif d’entreprise : en présence d’un délégué syndical, il doit être signé, d’une part, par l’employeur ou son représentant, et d’autre part, par une ou plusieurs organisations syndicales ayant recueilli plus de 50% des suffrages exprimés. Si cette condition n'est pas re ...
+    "reponse": "<p>Les conditions de validité de l’accord de performance collective sont identiques à celles de tout accord collectif d’entreprise : en présence d’un délégué syndical, il doit être signé, d’une part, par l’employeur ou son représentant, et d’autre part, par une ou plusieurs organisations syndicales ayant recueilli plus de 50% des suffrages exprimés. Si cette condition n'est pas re ...
```
Strange `’`...
Should be a `'` I think

jrduscher committed:
Update faq.json

fichier valide sur jsonlint+ ajout d'une QR

jrduscher committed:
Update faq.json

Ma période d'essai peut elle être renouvelée ? (Boulangerie): modification de contenu suite à GT

jrduscher commented:
le fichier dans sa dernière version passe dans jsonlint.

lionelB reviewed (approved):
merci !

lionelB committed:
fix(data): format using prettier

lionelB committed:
fix: remove ascii simple quote

douglasduteil reviewed (approved):
You work @lionelB :+1:
